src/LazyOperators/algebra.jl

```diff
@@ -65,6 +66,13 @@ Base.:/(a, ::NullOperator) = error("Division by an AbstractNullOperator is not a
 Base.:/(a::NullOperator, b) = a
 Base.:/(::NullOperator, ::NullOperator) = NullOperator()
 
+# For binary `^`:
+Base.:^(a, ::NullOperator) = error("Undefined")
```

Comment:
why defining a method that returns error rather than not defining anything and let julia raise an error?

Reply:
I think that if we don't raise an error explicitly we will forget that the method was not defined on purpose.